app/__init__.py

```diff
@@ -1,3 +1,12 @@
+from sqlalchemy.ext.declarative import declarative_base
```

Comment:
use local import like how I did it for the rest of the files.
Trying learning by observing

Reply:
Local import only for what can be imported locally